Orange/widgets/visualize/owscatterplot.py

```diff
@@ -322,9 +322,7 @@ def set_data(self, data):
             self.init_attr_values()
         self.vizrank.initialize()
         self.vizrank.attrs = self.data.domain.attributes if self.data is not None else []
-        self.vizrank_button.setEnabled(
-            self.data is not None and self.data.domain.class_var is not None
-            and len(self.data.domain.attributes) > 1 and len(self.data) > 1)
+        self._vizrank_button_enabled()
```

Comment:
I suppose that the bug that you are fixing here is that the button was not enabled when it should have been. This shouldn't be "fixed" so that the button is always enabled. These conditions were correct, so the bug -- the reason that this line was not executed -- must be somewhere else.

You should find the actual error and fix it instead of removing the code that (I suppose) works.